cpp/benchmarks/io/parquet/parquet_reader_benchmark.cpp

```diff
@@ -101,8 +109,17 @@ void BM_parq_read_varying_options(benchmark::State& state)
   auto const view = tbl->view();
 
   std::vector<char> parquet_data;
+  auto table_meta = cudf::io::table_input_metadata(view);
+  // Precision is required for decimal columns but the value doesn't affect the performance
+  for (cudf::size_type c = 0; c < view.num_columns(); ++c) {
```

Comment:
```suggestion
  for (auto c = 0; c < view.num_columns(); ++c) {
```

This might read easier.